k6/test.js

```diff
@@ -0,0 +1,18 @@
+import http from 'k6/http';
+import { sleep, check } from 'k6';
+
+export let options = {
+  vus: 10,  // 동시 접속 사용자
+  duration: '30s', // 테스트 지속 시간
+};
+
+export default function () {
+  let res = http.get('http://localhost:8080/movies?theater_id=2');
```

Comment:
```suggestion
const theaterId = 2;
const BASE_URL = 'http://localhost:8080';
http.get(`${BASE_URL}/movies?theater_id=${theaterId}`);;
```

쿼리 파라미터와 BaseURL은 이와 같이 변수로 분리해서 사용 할수 있어요!